### ISSUE TITLE
Segmentation fault when interaction is not present

### ISSUE THREAD

**Issue by breecummins** [shaunharker/DSGRN#49]
The method "interaction" of DSGRN.Network accepts two node indices and returns True if there is an activating edge between the two and False if there is a repressing edge. The method returns a segmentation fault if the interaction is absent. Either an informative error or a third value would be helpful.

**Comment by shaunharker** [comment]
Agreed, perhaps in the C++ code base it should throw an exception, and in the python wrappings the exception could be caught (something like https://pybind11.readthedocs.io/en/stable/advanced/exceptions.html) so a python "None" value could be returned.
